enterprise/server/cmd/goinit/main.go

```diff
@@ -156,7 +157,12 @@ func startDockerd(ctx context.Context) error {
 
 	log.Infof("Starting dockerd")
 
-	cmd := exec.CommandContext(ctx, "dockerd")
+	args := []string{}
+	if enableTCP {
```

Comment:
in `waitForDockerd` below, should probably also check `docker ps` using the TCP port if enabled, otherwise the unix socket might be ready but the TCP port might not be.

Reply:
Done